CHANGELOG.md

```diff
@@ -10,10 +10,11 @@ Full documentation for hipCUB is available at [https://rocm.docs.amd.com/project
 
 ### Changed
 
+* Updated the default value for the `-a` argument from `rmake.py` to `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`.
```

Comment:
What was the old default value?

```suggestion
* Changed the default value of `rmake.py -a` to `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`.
```

Reply:
It was the same but without gfx1200,gfx1201 and gfx1151

Reply:
Ah OK! Let me rewrite this then.